pyproject.toml

```diff
@@ -15,7 +15,8 @@ dependencies = [
     "langfuse>=2.38.2",
     "selenium>=4.0.0",
     "beautifulsoup4>=4.9.3",
-    "pyshadow<=0.0.5"
+    "pyshadow<=0.0.5",
+    "tomli>=2.0.1",
```

Comment:
is this needed - don't see it used in here

Reply:
it's used in check_license which was failing for me due to the lack of this lib